.gitattributes

```diff
@@ -0,0 +1,7 @@
+# This way, the files would be available in the repository but it would not be downloaded when the package is required by another project.
+/tests             export-ignore
+.gitattributes     export-ignore
+.gitignore         export-ignore
+.travis.yml        export-ignore
+Dockerfile         export-ignore
+phpunit.xml        export-ignore
```

Comment:
Nice! could you add a newline at the end of the file?

Reply:
Of course! I forgot... :sweat_smile: